docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+services:
+  db:
+    image: postgis/postgis:15-3.3
+    container_name: biotablero-postgis-db
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_INITDB_ARGS: "--auth-local=scram-sha-256"
+      POSTGIS_GDAL_ENABLED_DRIVERS: ENABLE_ALL
+      POSTGIS_ENABLE_OUTDB_RASTERS: "true"
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./app/db/initdb:/docker-entrypoint-initdb.d
+    networks:
+      - app-network
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    command: >
+      bash -c "apt-get update &&
+               apt-get install -y postgresql-15-postgis-3 postgresql-15-postgis-3-scripts &&
+               docker-entrypoint.sh postgres"
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: biotablero-search-backend
+    command: uvicorn app.main:app --host 0.0.0.0 --port 8000 --reload
+    ports:
+      - "8000:8000"
+    environment:
+      DATABASE_URL: postgresql+psycopg2://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
+    depends_on:
+      db:
+        condition: service_healthy
+    networks:
+      - app-network
```

Comment:
I think dockerizing the backend app is outside the scope of this task. Did you include it for any specific reason? Does Alembic require it?